lib/modules/manager/vendir/types.ts

```diff
@@ -1,36 +0,0 @@
-import type { HostRule } from '../../../types';
```

Comment:
was it already unused? move to separate PR

Reply:
Yes. It's removal was supposed to be contained in the previous vendir PR, but that change got missed.

Reply:
Will move to a separate PR